content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -213,10 +218,7 @@ When managing the scale of a group of replicas using the Horizontal Pod Autoscal
 it is possible that the number of replicas keeps fluctuating frequently due to the
 dynamic nature of the metrics evaluated. This is sometimes referred to as *thrashing*.
 
-Starting from v1.6, a cluster operator can mitigate this problem by tuning
-the global HPA settings exposed as flags for the `kube-controller-manager` component:
-
-Starting from v1.12, a new algorithmic update removes the need for the
+Starting from v1.12, a new algorithmic update removes the need for an
```

Comment:
The olded supported version is v1.12; if you want, you can drop mention of upscale delay entirely.